login.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>magniZent Store</title>
+    <link
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="./style/header.css" />
+    <link rel="stylesheet" href="./style/colors.css" />
+    <link rel="stylesheet" href="./style/reset.css" />
+    <link rel="stylesheet" href="./style/utility.css" />
+    <link rel="stylesheet" href="./style/login.css" />
+  </head>
+
+  <body>
+    <div class="main-login-box">
+      <div class="header-container">
+        <div class="header">
+          <div
+            class="logo flex-container flex-center"
+            style="align-items: baseline; gap: 2rem"
```

Comment:
inline style?

Reply:
Thank you for pointing it out. I forgot to remove it.